src/index.html

```diff
@@ -11,7 +11,226 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="form-block">
+
+        <legend>Personal information</legend>
```

Comment:
do not leave blank lines between parent and child tags (at all similar places)

```suggestion
    >
      <fieldset class="form-block">
        <legend>Personal information</legend>
```